modules/library.js

```diff
@@ -0,0 +1,108 @@
+const list = document.getElementById('book_list');
+const AddBook = document.getElementById('AddBook');
+
+export default class Library {
```

Comment:
- [ ] Please use the ES6 arrow function.

Reply:
Thank you for that suggestion, but I don't understand what you want me to do exactly. How can I use the arrow function for declaring a class or a variable?

Reply:
hello again, sorry to reply late, you see line number 4 you have a class library that's what I  want you to look at.

Reply:
I used const it resulted in an error. This is js class I don't see where I could use an arrow function.

Reply:
With all due respect, can you please check if this change is necessary?